docs/content/docs.md

```diff
@@ -1208,7 +1208,7 @@ This can be useful as an additional layer of protection for immutable
 or append-only data sets (notably backup archives), where modification
 implies corruption and should not be propagated.
 
-### -i / --interactive {#interactive}
+### -i, --interactive {#interactive}
```

Comment:
Found this more consistent with the format used for other flags with short- and long form, in this page. E.g "-I, --ignore-times", "-V, --version", "-v, -vv, --verbose", "-u, --update", "-q, --quiet", "-P, --progress".